app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+from typing import List
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connect = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connect.execute(
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) "
+            "VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connect.commit()
+
+    def all(self) -> List[Actor]:
+        actor_cursor = self._connect.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actor_cursor]
+
+    def update(self,
```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one, the closing bracket is also written on a new line

Reply:
done